### ISSUE TITLE
panic: runtime error: slice bounds out of range

### ISSUE THREAD

**Issue by bzz** [src-d/datasets#41]
Our official example from the documentation https://github.com/src-d/datasets/tree/master/PublicGitArchive/pga#downloading-siva-files results in a `panic`.

Steps to reproduce:
```
pga list -u github.com/src-d/ -f json | jq -r 'select(.fileCount > 50) | .sivaFilenames[]' | pga get -i -o repositories
```

Expected: downloading files to `./repositories` starts.
Actual: panic.

May be relevant to `-i` option, as noted in https://github.com/src-d/datasets/issues/33#issuecomment-371986544.

*Update*: 

`echo "cce947b98a050c6d356bc6ba95030254914027b1.siva" | pga get -i`
results in the same 

```
downloading siva files by name from stdin
filter flags will be ignored
panic: runtime error: slice bounds out of range
```

**Comment by bzz** [comment]
Initial fix in #41

**Comment by bzz** [comment]
fixed on latest master
